support/metrics/labels.go

```diff
@@ -0,0 +1,20 @@
+package metrics
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	// The common label for monitoring in HyperShift
+	// Namespaces with this label will be actively monitored by the observability operator
+	HyperShiftMonitoringLabel = "hypershift.openshift.io/monitoring"
```

Comment:
I forgot to mention in the main PR but I'd expect this annotation to be named according to its consumer e.g. `hypershift.openshift.io/obo-monitoring`
That can be a follow up though.

Reply:
While I understand your point, from a platform perspective, this label is/will-be standardized indicating the namespace is being monitored by HyperShift's default monitoring stack (OBO) hence I do not see a need for explicitly defining a consumer.
If we go that route, we'll have to make changes in https://issues.redhat.com/browse/OSD-15906 as well which is already in production.